en/db-migrations.md

```diff
@@ -203,7 +203,7 @@ Database migrations support the following database column types:
 
 <a name='writing'></a>
 ## Writing Migrations
-Migrations aren't only designed to 'morph' table. A migration is just a regular PHP class so you're not limited to these functions. For example after adding a column you could write code to set the value of that column for existing records. For more details and examples of individual methods, check the [database component](/4.0/en/db).
+Migrations aren't only designed to 'morph' table. A migration is just a regular PHP class so you're not limited to these functions. For example after adding a column you could write code to set the value of that column for existing records. For more details and examples of individual methods, check the [database component](/[:version:]/[:language:]/db-layer).
 
```

Comment:
This will not work since there is no replace method for the documents to replace the version and the language in languages.

The best way to address this for all languages is to change the link to `/db-layer`. Since you are already in the say `/4.0/dk-dk/` folder that would work